lib/cext/ruby.h

```diff
@@ -218,6 +218,7 @@ void rb_check_safe_obj(VALUE object);
 bool SYMBOL_P(VALUE value);
 
 // Constants
+#define RUBY_VM 1
```

Comment:
Used in `pg` https://github.com/ged/ruby-pg/blob/1f274f68c16f1af1c642db1b9d3d28aef169dc84/ext/pg.h#L64